sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -161,9 +161,14 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
    *
    * This is only used for Common Table Expressions.
    */
-  override def visitNamedQuery(ctx: NamedQueryContext): SubqueryAlias = withOrigin(ctx) {
-    SubqueryAlias(ctx.name.getText, plan(ctx.query))
-  }
+  override def visitNamedQuery(ctx: NamedQueryContext): SubqueryAlias =
+    withOrigin(ctx) {
```

Comment:
nit. Shall we keep the original form like the following? That reduces both the diff of patch and the depth of indentation of the function body.
```scala
override def visitNamedQuery(ctx: NamedQueryContext): SubqueryAlias = withOrigin(ctx) {
```

Reply:
thanks, reverted to the original form